.github/workflows/container-reusable.yml

```diff
@@ -0,0 +1,82 @@
+name: Refresh container image
```

Comment:
This is basically an action, isn't it? Maybe we could keep actions in `.github/actions/` or something like that and give this file a normal name. The `container-reusable` name doesn't say anything useful.

Reply:
If you have some better idea for the file name, that's certainly doable. How about `rebuild-container-action-reusable.yml`?

For the `name:` part, that is reused in the UI, so I don't want to make it too long, or we won't know what the individual runs are. In fact, it would be good to make it even shorter, the auto-suffix ` (rpm)` is truncated already on the left side job list.

As to another directory, unfortunately the file [must be exactly where it is](https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow).

Rant mode on: The terminology and capability mix is pretty weird. Normally, you have a _workflow_ which consists of _jobs_ which have _steps_. A step can be either an _action_ call, or a bash script. The files you put into `.github/workflows` define whole workflows, each complete with its jobs and steps, no preprocessing, no includes, no hierarchy. An action must be a separate repo, which provides some separation, but it does not feel made to be a separation measure, it mostly enables easy consumption of third party actions. A reusable workflow is a bit of a hack within that system, it's an almost-workflow, which does not run on its own but is invoked as a job (IIRC). As a corollary, it's a job, not a step, which means you can't combine it with the bash kind of step, you have to create a chain of jobs with dependencies. And you can't use matrix/strategy in workflows that call reusable workflows, only inside the reusable workflows themselves... it's all just lovely.

Reply:
Now it's `container-rebuild-action.yml`...